apps/web/app/(app)/gardens/[chain]/[garden]/[community]/page.tsx

```diff
@@ -0,0 +1,201 @@
+    try {
+      const response = await fetch("https://ipfs.io/ipfs/" + covenantIpfsHash);
+      const json = await response.json();
+      covenant = typeof json.covenant === "string" && json.covenant;
+    } catch (error) {
+      console.log(error);
+    }
+  }
+
+  const activePools = strategies?.filter((strategy) => strategy?.isEnabled);
+
+  return (
+    <div className="page-layout">
+      <header className="section-layout flex flex-row items-center gap-10">
+        <div>
+          <Image
+            src={commImg}
+            alt={`${communityName} community`}
+            className="h-[180px]"
+            height={180}
+            width={180}
+          />
+        </div>
+        <div className="flex flex-1 flex-col gap-2">
+          <div>
+            <h2>{communityName}</h2>
+            <EthAddress address={communityAddr as Address} />
+          </div>
+          <div className="flex flex-col gap-2">
+            <Statistic label="members" count={members?.length ?? 0} />
+            <Statistic
+              label="pools"
+              icon={<RectangleGroupIcon />}
+              count={activePools.length ?? 0}
+            />
+            <Statistic label="staked tokens" icon={<CurrencyDollarIcon />}>
+              <DisplayNumber
+                number={[BigInt(communityStakedTokens), tokenGarden?.decimals]}
+                compact={true}
+                tokenSymbol={tokenGarden?.symbol}
+              />
+            </Statistic>
+          </div>
+        </div>
+        <div className="flex flex-col gap-4">
+          {/* <div className="flex flex-col gap-2">
+            <Statistic
+              label="registration amount"
+              count={registerStakeAmount}
+            />
+            <Statistic label="community fee" count={communityFee} />
+          </div> */}
+          <RegisterMember
+            tokenSymbol={tokenGarden?.symbol ?? ""}
+            communityAddress={communityAddr as Address}
+            registerToken={tokenAddr as Address}
+            registerTokenDecimals={tokenGarden?.decimals}
+            membershipAmount={registerStakeAmount}
+            protocolFee={protocolFee}
+            communityFee={communityFee}
+          />
+        </div>
+      </header>
+      <IncreasePower
+        communityAddress={communityAddr as Address}
+        registerToken={registerToken as Address}
+        tokenSymbol={tokenGarden?.symbol ?? ""}
+        registerTokenDecimals={tokenGarden?.decimals as number}
+        registerStakeAmount={BigInt(registerStakeAmount)}
+      />
+      <section className="section-layout flex flex-col gap-10">
+        <header className="flex justify-between">
+          <h2>Pools</h2>
+          <FormLink href={`/gardens/${chain}/${tokenAddr}/${communityAddr}/create-pool`} label="Create Pool" />
+        </header>
+        <div className="flex flex-col gap-4">
+          <h4 className="text-secondary-content">
+            Funding pools ( {fundingPools.length} )
```

Comment:
I think its better keep no space between parenthesis like in figma: 
![image](https://github.com/1Hive/gardens-v2/assets/13890618/c59d9746-33e9-400a-a387-d7b255984092)

Reply:
totally missed it, now it is fixed